pam/pam.go

```diff
@@ -222,6 +224,10 @@ func pam_sm_authenticate(pamh *C.pam_handle_t, flags, argc C.int, argv **C.char)
 			case "allowed":
 				fmt.Printf("Welcome:\n%s\n", iaResp.UserInfo)
 				return C.PAM_SUCCESS
+			case "cancelled":
```

Comment:
cancelled is not a returned value. so no case here. Those are the returned value from the broker, not the UI.

Reply:
Not sure what you mean. I've added it as a return value in the IsAuthorized functions and [enabled it here](https://github.com/ubuntu/authd/blob/5f9acaac192e5f76cd9781a4446045f34b841bba/internal/brokers/broker.go#L115).
It feels weird to have the method returned "denied" if it was cancelled, isn't it? I thought it would be better to have a different access level returned after a cancel so that we could properly handle it, instead of erroring it out (as this would trigger some decorates and other stuff in the flow, no?) and handling the error.

Reply:
Right, that makes sense in some way to separate it from denied. In most case, we will ignore the value anyway, so ok.